contents/docs/session-replay/_snippets/ios-logging.mdx

```diff
@@ -0,0 +1,96 @@
+As console logs can contain sensitive information, we do not capture these logs automatically. 
+You can enable this feature from your client-side by setting `captureLogs = true` in our [iOS SDK config](/docs/session-replay/installation?tab=iOS#step-two-enable-session-recordings-in-your-project-settings).
+
+```ios_swift
+let config = PostHogConfig(apiKey: <ph_project_api_key>, host: <ph_client_api_host>)
+config.sessionReplay = true
+config.sessionReplayConfig.captureLogs = true // TIP: Needs to be configured in code. Project settings won't have an effect.
+
+PostHogSDK.shared.setup(config)
+```
+
+> **Important:** Individual console logs are truncated at 2000 characters. The rest of the log is not sent to PostHog. When truncating a log message we add ...[Truncated] to the message.
+
+### Custom log processing and sanitization
+
+PostHog iOS SDK allows you to customize how log messages are processed and categorized by providing a custom `processConsoleOutput` handler. 
+This handler receives raw console output and returns an array of processed log entries with their respective severity levels:
+
+```ios_swift
+let config = PostHogConfig(apiKey: <ph_project_api_key>, host: <ph_client_api_host>)
+
+// Custom log processing and sanitization
+config.sessionReplayConfig.captureLogsConfig.processConsoleOutput = { output in
+
+    // Skip trace-level logs
+    guard !output.contains("TRACE") else {
+        return []
+    }
+
+    // Determine log level based on message content
+    let level: PostHogConsoleLogLevel = {
```

Comment:
I'd remove the `Console` name part here because of https://github.com/PostHog/posthog.com/pull/11473/files#r2089024840